vitrina/smart_contracts/permissions.py

```diff
@@ -1,15 +1,20 @@
+from vitrina.orgs.models import Organization
 from vitrina.smart_contracts.services import get_agreements
 from vitrina.users.models import User
 from vitrina.smart_contracts.models import Agreement
 from vitrina.projects.models import Project
 from django.contrib.auth.models import AnonymousUser
 
 
-def can_create_agreements(user: User, project: Project) -> bool:
-    if project.organization:
-        return project.organization == user.viisp_organization and user.is_representative_of(project.organization, True)
+def _is_viisp_authenticated_and_representative(user: User, organization: Organization | None) -> bool:
+    """Helper to find out if a user is VIISP authenticated and a representative of an organization."""
+    if not organization:
+        return False
 
-    return False
+    is_viisp_authenticated = organization == user.viisp_organization
+    is_representative = user.is_representative_of(organization, True)
+
+    return is_viisp_authenticated and is_representative
```

Comment:
```suggestion
    return is_viisp_authenticated and user.is_representative_of(organization, True)
```

Lazy better for performance. If `is_viisp_authenticated` is `False`- second part of the statement won't be executed.

Reply:
I like the readability less, but it is a nice suggestion, fixed it, thanks:)